src/test/java/org/openstreetmap/atlas/tags/HighwayTagTestCase.java

```diff
@@ -0,0 +1,43 @@
+package org.openstreetmap.atlas.tags;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openstreetmap.atlas.tags.annotations.validation.Validators;
+import org.openstreetmap.atlas.utilities.testing.TestTaggable;
+
+/**
+ * Test for {@link org.openstreetmap.atlas.tags.HighwayTag}
+ *
+ * @author v-garei
+ */
+public class HighwayTagTestCase
+{
+
+    @Test
+    public void highwayTagTests()
+    {
+        final Taggable emergencyBayTaggable = Taggable.with(HighwayTag.KEY, "emergency_bay");
```

Comment:
Here for the sake of searchability, I would replace `"emergency_bay"` with `HighwayTag.EMERGENCY_BAY.toString()`... Unless what you meant to test here is that the enum is properly spelled.

Reply:
you right you right!